src/pool/spawn.rs

```diff
@@ -142,11 +208,17 @@ impl<T: TaskCell + Send> QueueCore<T> {
 /// dropped.
 pub struct Remote<T> {
     pub(crate) core: Arc<QueueCore<T>>,
+    // Only used to protect the scale_workers method to avoid multiple threads
```

Comment:
Can we define the `Mutex` in the core? So we don't need an extra `Arc`.

Reply:
Fixed, thanks!